src/js/components/navigation.js

```diff
@@ -0,0 +1,63 @@
+import { newLink, newContainer } from './helpers';
+
+const createLinks = (dataItems) => {
+  const arrayLinks = dataItems.map((item) => {
+    const menuLink = newLink(item.href, item.text, ['heading', 'heading-3', 'nav__menu-link']);
+    const menuItem = newContainer('li', [menuLink], ['nav__menu-item']);
+    return menuItem;
+  });
+  return arrayLinks;
+};
+
+const mainNavigation = (data) => {
+  // Create menu button
+  const menuButton = document.createElement('button');
+  menuButton.classList.add('button', 'nav__button-menu');
```

Comment:
The class `button` is not necessary because you are applying the styles directly to the tag